src/App.js

```diff
@@ -29,37 +29,94 @@ const App = () => {
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
-
+  // const [winner, setWinner] = useState('');
+  const [currentPlayer, setCurrentPlayer] = useState(Player1);
   // Wave 2
   // You will need to create a method to change the square
   //   When it is clicked on.
   //   Then pass it into the squares as a callback
 
+  const onClickSquare = (id) => {
+    console.log(id);
+    const newSquares = [...squares];
+    if (checkForWinner() === null) {
```

Comment:
The `!` (not) operator can also be used to check if a value is _falsy_

```suggestion
    if (!checkForWinner()) {
```